requirements.txt

```diff
@@ -1,2 +1,3 @@
 pandas==0.20.3
 beautifulsoup4==4.6.0
+requests==2.18.3
```

Comment:
I've also added it to the setup.py file as a requirement before install the govukurllookup package, so we may not even need it here.